src/cursor/cur_backup.c

```diff
@@ -223,11 +224,12 @@ __backup_start(WT_SESSION_IMPL *session,
 	WT_DECL_RET;
 	WT_FSTREAM *srcfs;
 	const char *dest;
-	bool exist, log_only, target_list;
+	bool exist, is_dup, log_only, target_list;
 
 	conn = S2C(session);
 	srcfs = NULL;
 	dest = NULL;
+	is_dup = other != NULL;
```

Comment:
As far as I can tell, we're not actually using `other` for any purpose in this function, why not pass `other != NULL` to `__backup_start()`, as a boolean `is_dup`?

Reply:
Good idea. Fixed.